config/jest/jest.config.js

```diff
@@ -1,6 +1,7 @@
-import 'babel-polyfill'
-import Adapter from 'enzyme-adapter-react-16'
 import Enzyme from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+import "@babel/polyfill"
```

Comment:
usually the pollyfill be the first import in any file. Does this still work as intended?

Reply:
this is what I get when save file our importing formatter, format it on this way.